patchwerk-sim.py

```diff
@@ -39,10 +26,16 @@ def total_plus_heal(plus_heal, healclass):
 import random
 import statistics
 import sys
+import os
```

Comment:
let's keep imports in alphabetical order since this file will probably become bigger as we add features